index.html

```diff
@@ -33,9 +36,149 @@
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+          
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item ">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+           
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+    </div>
+</section>
+<section>
+   
+        <div class="about-me">
+            
+              <!-- <img class="right" src="./right.png" alt="image">   -->
+            <h2 class="about">About Me</h2>
+            
+            <p class="about-myself"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+               
+                <button class="get-my-resume"><p class="resume">Get My Resume</p></button>
+                <!-- <img class="left" src="./left.png" alt="image"> -->
+        </div>
+         <div class="grid-container-another">
+        <div class="grid-item-another">
+            <img class="skills languagelogo" src="./icon-languages@2x.png" alt="language logo">
+            <h3 class="language">Languages</h3>
+            
+                <ul class="mini-flex-container">
+                <li class="cli mini-item">Javascript</li>
+                 <li class=" mini-item">HTML</li>
+                 <li class=" mini-item">Ruby</li> 
+                 <li class=" mini-item">CSS</li> 
+                 </ul> 
+             
+        </div>
+        <div class="grid-item-another">
+            <img class="skills framelogo" src="./icon-frameworks@2x.png" alt="framework logo">
+            <h3 class="language">Frame Work</h3>
+         
+                <ul class="mini-flex-container">
+                <li class="cli mini-item">React.js</li>
+                 <li class=" mini-item">Rspec</li>
+                 <li class=" mini-item">capybara</li> 
+                </ul> 
+             
+        </div>
+        <div class="grid-item-another">
+            <img class="skills skillslogo" src="./icon-skills@2x.png" alt="skills logo">
+            <h3 class="language">Skills</h3>
+            
+                <ul class="mini-flex-container">
+               <li class="cli mini-item">CLI</li>
+                <li class="web mini-item">version control</li>
+                <li class="api mini-item">API design</li>
+                </ul>  
+            
```

Comment:
- [ ] Jist like the previous reviewer suggested, You need to make sure that your technology section looks like the given design as its part of the main requirement on the milestone that your project replicated the Figma design. I'll suggest you take a look at this section and update the margins and spacing. :smile: :small_airplane: 

Your design:
![design-](https://user-images.githubusercontent.com/107632780/235317254-48c648f2-6fca-4348-b4fc-7013b17d4f81.png)

Figma design:
![Screenshot from 2023-04-29 18-55-33](https://user-images.githubusercontent.com/107632780/235317265-e72adc30-0f1f-4724-becc-7dafb350def1.png)

Reply:
hey @V-Blaze ,
I added and re-correct the errors which you provided me. kindly check them pls